MMA Removal Utility/README.md

```diff
@@ -262,6 +337,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
 | project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
 virtualMachinesWithMMAExtensions
 ```
+
+For VMSS(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceId contains "microsoft.compute/virtualmachinescalesets"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend  OSType = tostring(parse_json(Metadata_s).OSType)
+| project VMResourceID = ResourceId, OSType;
+let virtualMachinesMMAExtensions = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VMExtension" and Source_s =~ "AzTS_05_VMExtensionInventoryProcessor"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend VMResourceID = tolower(substring(ResourceId,0,indexof(ResourceId, '/', 0, -1, 9 )))
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| where ExtensionType =~ "OmsAgentForLinux" or ExtensionType =~ "MicrosoftMonitoringAgent"
+| project ResourceId, VMResourceID, ExtensionType, ExtensionResourceId = ResourceId;
+let virtualMachinesWithMMAExtensions = virtualMachines
+| join kind=inner  (virtualMachinesMMAExtensions) on VMResourceID
+| project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
+virtualMachinesWithMMAExtensions
+```
+
+For Azure Arc Server(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
```

Comment:
Same as above query, contains check and variable naming

Reply:
Done